homeassistant/components/stream/worker.py

```diff
@@ -468,6 +476,10 @@ def stream_worker(
     # Some audio streams do not have a profile and throw errors when remuxing
     if audio_stream and audio_stream.profile is None:
         audio_stream = None
+    stream_state.diagnostics.set_value("container_format", container.format.name)
+    stream_state.diagnostics.set_value("video_stream", video_stream.name)
```

Comment:
Maybe rename these video_codec and audio_codec?

Reply:
Yes, much better, thanks.